tools/crm_resource_print.c

```diff
@@ -776,7 +776,11 @@ static void
 add_resource_name(pcmk_resource_t *rsc, pcmk__output_t *out)
 {
     if (rsc->children == NULL) {
-        out->list_item(out, "resource", "%s", rsc->id);
+        /* Sometimes PCMK_XE_RESOURCE might act as a PCMK_XA_NAME instead of an
```

Comment:
good to go back and check whether, perhaps, `--xml-simple-list` is forced for all command lines that lead to this point